modules/stayrtr.nix

```diff
@@ -14,12 +14,16 @@ in
   };
 
   config = lib.mkIf cfg.enable {
+    systemd.services.bird2 = {
+      after = [ "dn42-stayrtr.service" ];
```

Comment:
Do we also need requires here?

Reply:
Idk, bird fails to connect to the rtr server because it probably starts before it otherwise and therefore is not able to boot correctly. It tries a second time but with a bit of delay. With this after the routes are installed faster after the device booted.